docs/usage/database_containers.md

```diff
@@ -53,40 +53,40 @@ _N.B:_
 * _TC needs to be on your application's classpath at runtime for this to work_
 * _For Spring Boot you need to specify the driver manually `spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver`_
 
-**Original URL**: `jdbc:mysql://somehostname:someport/databasename`
+**Original URL**: `jdbc:mysql:5.7.22://somehostname:someport/databasename`
 
 Insert `tc:` after `jdbc:` as follows. Note that the hostname, port and database name will be ignored; you can leave these as-is or set them to any value.
 
 ### JDBC URL examples
 
-#### Simple Testcontainers JDBC driver usage
+#### DEPRECATED: Simple Testcontainers JDBC driver usage
 
 `jdbc:tc:mysql://somehostname:someport/databasename`
 
-*(Note: this will use the latest version of MySQL)*
+*(Note: this will implicitly use the `latest` version of MySQL. Using `latest` is risky and Testcontainers will soon require an explicit tag name to be provided)*
```

Comment:
Maybe we should implicitly set it to 5 if no version was provided?

Reply:
Perhaps - I think I might sleep on it. Both approaches have tradeoffs, so I guess it's just a case of which one we think is least troublesome for users.

Reply:
True. Just thought that given the fact that the latest MySQL broke our own tests, a lot of users will also be affected, especially the ones who use the JDBC url approach :)

Reply:
I've decided 😆 
Updated following your suggestion. We're now using a fixed version (which is similar to current latest) if there is no tag provided in the JDBC URL.